CMakeLists.txt

```diff
@@ -125,6 +129,31 @@ elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "Darwin")
 	include_directories(SYSTEM "include/support/osx")
 endif()
 
+# If we're on windows, do a check for what compiler we're using (so we can link the appropriate libraries)
+if(WIN32)
+    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU") # We're dealing with MinGW etc
+        set(Boost_USE_STATIC_LIBS ON)
+        set(BOOST_INCLUDEDIR ${LIB_ROOT_DIR}/boost/boost)
```

Comment:
`LIB_ROOT_DIR` should be a documented CMake variable that appears in the configurator (.e.g `cmake-gui` or `ccmake`).

Reply:
I've passed it in with -D in the .bat file, does that not create the variable? It needs to be a defined CMake one?